src/types/checkers.ts

```diff
@@ -9,7 +9,7 @@ import {
 export const isBlockOneLevelDeep = (
   data: Data<DownloadableContent>
 ): data is Data<BlockOneLevelDeep> =>
-  !!(data as Data<BlockOneLevelDeep>).attributes.Document
+  !!(data as Data<BlockOneLevelDeep>).attributes.Slides
```

Comment:
Why this change?

Reply:
It seemed `Document` does not always exist on Blocks, whereas the `Slides` array does

Reply:
Ah right, that's true! Nice